### PR TITLE
typo fix s/mode/i_mode/

### PR DESCRIPTION
**Pull request by cbkadal**
And add '+' to Makefile.

This is my first "pull request" ever. Wish me luck for not screwing anything.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -6,7 +6,7 @@ KDIR ?= /lib/modules/$(shell uname -r)/build
 MKFS = mkfs.simplefs
 
 all: $(MKFS)
-	make -C $(KDIR) M=$(PWD) modules
+	+make -C $(KDIR) M=$(PWD) modules
```
Comment (jserv) [inline Makefile:9]: Don't change this line.

Comment (cbkadal) [reply Makefile:9]: what should I do?

Comment (jserv) [reply Makefile:9]: Revert this change.

**Comment by jserv** [comment]
Close in favor of #27

**Pull request closed without merging**